cmd/bisync/log.go

```diff
@@ -29,16 +29,29 @@ func (b *bisyncRun) indent(tag, file, msg string) {
 		logf = fs.Logf
 	}
 
+	if b.opt.DryRun {
+		logf = fs.Logf
+		msg += " in dry-run mode"
+	}
+
 	if tag == "Path1" {
 		tag = Color(terminal.CyanFg, "Path1")
+	} else if tag == "Path2" {
```

Comment:
The idea here was to use different colors for Path1 and Path2, to make it easier to visually differentiate in the terminal. So I would suggest either removing this change or using a different color.

Reply:
ill change to green, if that does not work, I can easily just remove.